src/app/store/index.ts

```diff
@@ -0,0 +1,10 @@
+import {ActionReducerMap} from "@ngrx/store";
+import * as fromLayout from './reducers/login.reducers';
+
+export interface State {
+  login: fromLayout.State
+}
+
+export const reducers: ActionReducerMap<State> = {
+  login: fromLayout.reducer
+};
```

Comment:
Please move this file to `reducers` catalog and make reducer for `groups` actions

Reply:
Done